core/Symbols.cc

```diff
@@ -1112,6 +1112,65 @@ TypePtr Symbol::sealedSubclassesToUnion(const GlobalState &gs) const {
     return result;
 }
 
+// Record a required ancestor for this class of module
+void Symbol::recordRequiredAncestor(MutableContext ctx, SymbolRef ancestor, Loc loc) {
```

Comment:
Does this need to be `MutableContext`? It doesn't look like there's anything that requires more than `GlobalState &` in this method.

Reply:
Indeed, a non-const `GlobalState` is enough